Source/Paths/SvgQuadraticCurveSegment.cs

```diff
@@ -34,6 +36,7 @@ private static PointF CalculateSecondControlPoint(PointF controlPoint, PointF en
             return new PointF(x2, y2);
         }
 
+#if !NO_SDC

```

Comment:
@wieslawsoltes 

I think it surrounds `AddToPath`.

Reply:
It surrounds `AddToPath` and the helper function `CalculateControlPoint` used in `AddToPath`. Is that wrong?

Reply:
I think I will submit PR with splitting of all drawing code inside NO_SDC if/def into separate files. This should be easier to read and maintain also make next step closer for https://github.com/svg-net/SVG/issues/590

Reply:
Started work here https://github.com/svg-net/SVG/pull/934 @H1Gdev @mrbean-bremen your input would be appreciated

Reply:
That sounds good, thanks! I will have a look later today.